src/presets/cloudflare.ts

```diff
@@ -22,5 +22,15 @@ export const cloudflare = defineNitroPreset({
         JSON.stringify({ lockfileVersion: 1 }, null, 2)
       );
     },
+    "rollup:before"(nitro) {
```

Comment:
This hook seems to be too late. Do we have a way for preset to extend the config?

Reply:
Yes, preset is basically config preset we can add `wasm: {directImport: true }` to the preset. We shall introduce top level `wasm` option btw.